.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.13', '1.14']
+        go: ['1.14', '1.15']
```

Comment:
curious and feel free to not answer if busy. what would make you decide to extend the range to 1.16? and if so, would you lift the floor also?

Reply:
Well, 1.16 wasn't released when I made this change :)

Reply:
And yes, the intent is to have this track the currently supported Go versions, which are the current version and the previous version, so technically should be 1.15 and 1.16, now.